lib/active_admin/views/components/tabs.rb

```diff
@@ -15,17 +15,26 @@ def build(&block)
       end
 
       def build_menu_item(title, options, &block)
-        fragment = options.fetch(:id, title.parameterize)
+        fragment = options.fetch(:id, fragmentize(title))
+
         html_options = options.fetch(:html_options, {})
         li html_options do
           link_to title, "##{fragment}"
         end
       end
 
       def build_content_item(title, options, &block)
-        options = options.reverse_merge(id: title.parameterize)
+        options = options.reverse_merge(id: fragmentize(title))
         div(options, &block)
       end
+
+      private
+
+      def fragmentize(string)
+        result = string.parameterize
+        result = string.hash if result.blank?
```

Comment:
```suggestion
        result = Digest::MD5.hexdigest(string) if result.blank?
```

Reply:
Inconsistent string hash values would brake nothing in terms of navigation. Though using MD5 is definitely better.
P.S.
Thanks for the tip @timoschilling

Reply:
It will be break if you are on an tab and reload the page an get the request from a different server / instance.

You can reproduce it it:
- one an tab with hash value as id
- restart your server (if you use pow or puma local, you need to restart that too)
- reload the page and you will be on the first tap instatt of the hash value tab

Reply:
What about Base64 encoded string? It's shorter and more recognizable.

Reply:
No problem, we have Timo! :)

Reply:
By the way, I checked this on Chromium and we have this problem for all kind of tabs, and without restarting the server. It sounds like we need to update the current URL with the tab id.

But when we fix that, the issue would reproduce as Timo said, and his suggestion would fix it.

Reply:
> What about Base64 encoded string? It's shorter and more recognizable.

Yeah, that sounds even better! [`urlsafe_encode64`](https://ruby-doc.org/stdlib-2.5.3/libdoc/base64/rdoc/Base64.html#method-i-urlsafe_encode64) maybe?

Reply:
@deivid-rodriguez Sure 🙂